crates/torii/core/Cargo.toml

```diff
@@ -10,10 +10,12 @@ version.workspace = true
 
 [dependencies]
 anyhow.workspace = true
+async-graphql = { version = "5.0.8", features = [ "chrono", "dynamic-schema" ] }
```

Comment:
is this necessary? core shouldn't know about graphql

Reply:
Fixed on commit 6d5b936